Context.go

```diff
@@ -1,14 +1,14 @@
 package giu
 
 import (
-	"sync"
+    platform2 "github.com/AllenDang/giu/platform"
```

Comment:
platform2 is a bad name. Suggest to change to imguiPlatform for better understanding, so to renderer.